src/kokkos/plugin-SiPixelClusterizer/kokkos/SiPixelRawToClusterGPUKernel.cc

```diff
@@ -688,14 +688,11 @@ namespace KOKKOS_NAMESPACE {
         Kokkos::deep_copy(
             execSpace, Kokkos::subview(nModules_Clusters_h, 0), Kokkos::subview(clusters_d.moduleStart(), 0));
 
-#ifdef KOKKOS_BACKEND_SERIAL
-        const uint32_t threadsPerBlock = 1;
-#else
-        const uint32_t threadsPerBlock = 256;
-#endif
         const uint32_t blocks = ::gpuClustering::MaxNumModules;
+        Kokkos::TeamPolicy<KokkosExecSpace> teamPolicy(execSpace, blocks, Kokkos::AUTO());
```

Comment:
@jtchilders Could you double-check my changes here and in `gpuClustering.h` and `gpuClusterChargeCut.h`?